deploy/app.yaml

```diff
@@ -0,0 +1,101 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: bot-api
+  namespace: kscoutbot
+  labels:
+    app: bot-api
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: bot-api
+  template:
+    metadata:
+      labels:
+        app: bot-api
+    spec:
+      containers:
+        - name: bot-api
+          image: kscout/bot-api:dev-latest
+          env:
+            - name: APP_DB_USER
+              valueFrom:
+                configMapKeyRef:
+                  name: prod-db-config
```

Comment:
The database configuration is prefixed with `prod-` but none of the other secrets or config maps are, probably want to be consistent here.